File: SE5-P/ejer2.py
l1 = int(input("Ingrese lado 1: "))
l2 = int(input("Ingrese lado 2: "))
l3 = int(input("Ingrese lado 3: "))

if (l1 == l2 and l2 == l3):
    print("\nEQUILÁTERO")
elif (l1 == l2 or l2 == l3 or l1 == l3):
    print("\nISÓSCELES")
else:
    print("\nESCALENO")

File: SEM5-C/ejer2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SEM5_C
{
    internal class ejer2
    {
        static void Main(string[] args)
        {
            Console.Write("Ingrese lado 1: ");
            int l1 = int.Parse(Console.ReadLine());
            Console.Write("Ingrese lado 2: ");
            int l2 = int.Parse(Console.ReadLine());
            Console.Write("Ingrese lado 3: ");
            int l3 = int.Parse(Console.ReadLine());

            if (l1 == l2 && l2 == l3)
            {
                Console.WriteLine("\nEQUILÁTERO");
            }else if (l1 == l2 || l2 == l3 || l1 == l3)
            {
                Console.WriteLine("\nISÓSCELES");
            }else
            {
                Console.WriteLine("\nESCALENO");
            } 
        }
    }
}
